### PR TITLE
fix: context related errors resolved

### PR DESCRIPTION
**Pull request by vip-git**

### CODE REVIEW

File: email.android.js [hunk 1]
```diff
@@ -8,9 +8,9 @@ var fs = require("file-system");
 exports.available = function () {
   return new Promise(function (resolve, reject) {
     try {
-      var uri = android.net.Uri.fromParts("mailto", "eddyverbruggen@gmail.com", null);
+      var uri = android.net.Uri.fromParts("mailto", "frontend-licenties@essent.nl", null);
```
Comment (dennis-montana) [inline email.android.js]: I would change this to something not connected to a real essent address, something like john@doe.com.

**Review by dennis-montana — DISMISSED** [review]

File: email.android.js (hunk 1 above)
Comment (vip-git) [reply email.android.js]: i am planning to make it configurable so it is easy to change.

Comment (spike1292) [inline email.android.js]: not `regelneef@energiedirect.nl`?

Comment (vip-git) [reply email.android.js]: kept it blank as it worked without this

**Comment by dennis-montana** [comment]
Did you test the change to a blank email address on a device with and without email setup?

**Comment by vip-git** [comment]
Yes i have tested this on my local it works fine

**Review by dennis-montana — APPROVED** [review]

**Pull request merged**